pkg/crc/machine/bundle/metadata_test.go

```
@@ -188,6 +190,11 @@ func TestGetBundleType(t *testing.T) {
 }
 
 func TestVerifiedHash(t *testing.T) {
+	// skip the test when running in disconnected env
+	if disconnected, _ := strconv.ParseBool(os.Getenv("CRC_IN_DISCONNECTED_ENV")); disconnected {
+		t.Skip("Skipping test since running in disconnected env")
+	}
```

Since we are ignoring the error then why not just ?
```
if os.Getenv("CRC_IN_DISCONNECTED_ENV") == "true" {
 ...
}
```